cmd/ocm/login/cmd.go

```diff
@@ -329,7 +330,7 @@ func run(cmd *cobra.Command, argv []string) error {
 	cfg.Insecure = args.insecure
 
 	// Create a connection and get the token to verify that the crendentials are correct:
-	connection, err := cfg.Connection()
+	connection, err := ocm.NewConnection().Config(cfg).WithApiUrl(gatewayURL).Build()
```

Comment:
WithApiUrl looks redundant with that same URL also being set in the config, is there a particular use case it is needed in this spot?

Reply:
In this case the config is loaded but the URL is provided by the flag. 

In a case where someone's config is currently loaded with Staging but they pass in the `--url production` to the login command, this would ensure that the login command's `--url` flag is respected.

Reply:
Interesting, I would think that with `config.URL` being assigned on line 327 with the same gatewayURL and then used in the builder pattern, that it would be used to build with the updated flag value. Just to confirm, are you saying that will not be the case? I could be reading this wrong.

Reply:
Looking at it again I think that you're actually correct here. Let me make and test the change real quick!

Reply:
Worked just fine, great catch!